templates/kafka_client.conf.j2

```diff
@@ -26,8 +26,8 @@ security.protocol="SASL_PLAINTEXT"
 
 {% if cluster.sasl_protocol|upper == "PLAIN" %}
 sasl.jaas.config="org.apache.kafka.common.security.plain.PlainLoginModule required username='{{ cluster.username }}' password='{{ cluster.password }}';"
-{% elif cluster.sasl_protocol|upper == "SCRAM" %}
-sasl.jaas.config="org.apache.kafka.common.security.scram.ScramLoginModule required username='{{ cluster.password }}' password='{{ cluster.password }}';"
+{% elif cluster.sasl_protocol|upper|regex_search("^SCRAM") %}
```

Comment:
I am putting a `startwith` approach as SCRAM need to be SCRAM-SHA-256 or SCRAM-SHA-512

Reply:
Oh good catch. Good old copy paste. Hadn't gotten to testing a SCRAM install yet and whipped it from quick memory as a nicety.

Reply:
By this comment, are you going to be changing the REGEX for a STARTSWITH? Or is the REGEX you're answer to this? Either is good with me.